testsuite/forge-cli/src/main.rs

```diff
@@ -1450,8 +1447,8 @@ fn realistic_env_graceful_overload() -> ForgeConfig {
                     // overload test uses more CPUs than others, so increase the limit
                     // Check that we don't use more than 18 CPU cores for 30% of the time.
                     MetricsThreshold::new(18.0, 40),
-                    // Check that we don't use more than 5 GB of memory for 30% of the time.
-                    MetricsThreshold::new_gb(5.0, 30),
+                    // Check that we don't use more than 6 GB of memory for more than 10% of the time.
+                    MetricsThreshold::new_gb(6.0, 10),
```

Comment:
I read somewhere you'll do 7 GB. 6 GB is fine too, just wanted to double check this is intended

Reply:
I thought we were checking overall memory, but are checking RSS, which is a bit smaller.